Web_Development/FrontEnd/HTML_Commands/HML_Commands.md

```diff
@@ -1 +1,85 @@
-hi
\ No newline at end of file
+## Basic HTML Commands
+<hr>
+
+### The HTML tag: 
+The <html> tag points that the code is written in HTML and </html> tag states that it's the end of HTML code.
+
+### The head tag: 
+The head of an HTML code contains special information, for example: it's title. 
+```
```

Comment:
Change this to "  ```html  ", and do the same for the rest.

Reply:
changes done!

Reply:
Do the same for all the snippets, not just one.

Reply:
Also, do these changes: 
'''Add syntax highlighting to code snippets.
Add proper indentation to code.
Also, add some SS for better understanding if possible.'''

Reply:
please check!